scripts/dag_bootstrap_startup.sh

```diff
@@ -240,8 +245,13 @@ Error         = task_process/daemon.err.\$(Cluster).\$(Process)
 
 Queue 1
 EOF
-        # TODO - remove chmod
-        chmod 777 task_process/task_proc_wrapper.sh
+
+if [ "X$useHtcV2" != "X" ]; then
+  cat >> task_process/daemon.jdl << EOF
+environment = "useHtcV2=True"
+EOF
+fi
```

Comment:
thanks for notificing the missing indent.
I would have thought the code to be self-explanatory: if useHTCV2 is set in the environment then add a line in the submission JDL of the task_process so that it is set also in the environment of that job :man_shrugging: 

Anyhow I am adding that as a comment as well.